### ISSUE TITLE
Odoo `custom resource` shows undefined for fields

### ISSUE THREAD

**Issue by cgsmith** [n8n-io/n8n#4501]
**Describe the bug**
The fields after selecting custom resource are undefined which prevents the user from selecting which fields to populate

**To Reproduce**

1. Go to 'Odoo Node'
2. Click on 'Custom Resource'
3. Scroll down to 'Add Value'
4. See error after trying to select field

![image](https://user-images.githubusercontent.com/570018/199335254-1f685b17-3dd0-457c-ba98-e11f4e732590.png)

**Expected behavior**
Fields should be defined

**Environment (please complete the following information):**

- OS: Pop!_OS 22.04 LTS
- n8n Version 0.200.1
- Node.js Version 16.15.0
- Database system SQLite
- Operation mode own

**Additional context**
I fixed the issue and will be making a PR shortly. I confirmed locally. Are there any plans for creating unit tests?

**Comment by cgsmith** [comment]
Issue here is related  https://github.com/digital-boss/n8n-nodes-odoo/issues/2

**Comment by Joffcom** [comment]
Hey @cgsmith,

Does the PR you created break the lists for older versions of Odoo or does it work with them all?

**Comment by cgsmith** [comment]
Yes this should work with older versions. To be honest, this PR should fix a lot of issues with Odoo - not sure how it would have worked without this PR.

**Comment by Joffcom** [comment]
Hey @cgsmith 

Perfect, I noticed someone on the PR has said it doesn't work but I suspect it is a different issue.

I will aim to get it merged this week.

**Comment by Lanhild** [comment]
Hey @cgsmith @Joffcom 
https://github.com/n8n-io/n8n/pull/5803 fixes this issue.

cc linked PR also fixes https://github.com/n8n-io/n8n/issues/5241

**Comment by cgsmith** [comment]
:melting_face:
